deploy/charts/kube-oidc-proxy/values.yaml

```diff
@@ -54,6 +54,14 @@ tokenPassthrough:
   enabled: false
   audiences: []
 
+# To add extra impersonation headers
+# https://github.com/jetstack/kube-oidc-proxy/blob/master/docs/tasks/extra-impersonation-headers.md
+extraImpersonationHeaders:
+  clientIP: false
+  headers: #{}
```

Comment:
Not too sure this looks quite awkward, maybe we can just remove that line.

The `headers: #{}`, somehow makes me think it's a object rathen than a string